docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -21,25 +21,25 @@ For all tables below, the following convention is used:
 Interfaces
 ----------
 
-==============================  =============================================  =============
-Apache Cassandra                Version                                        Comments
-==============================  =============================================  =============
-CQL                             CQL spec 3.4.0, Native protocol v4 (C* 3.11)   More below
-------------------------------  ---------------------------------------------  -------------
+==============================  =====================================================  =============
+Apache Cassandra                Version                                                Comments
+==============================  =====================================================  =============
+CQL                             CQL spec |cql-version|, Native protocol v4 (C* 3.11)   More below
+------------------------------  -----------------------------------------------------  -------------
 Thrift                          Compatible with Cassandra 2.1
-------------------------------  ---------------------------------------------  -------------
-SSTable format (all versions)   3.11(mc / md ), 2.2(la), 2.1.8 (ka)            md format is supported in Scylla 4.3, 2021.1.0 and later
-------------------------------  ---------------------------------------------  -------------
-JMX                             3.11                                           More Below
-------------------------------  ---------------------------------------------  -------------
+------------------------------  -----------------------------------------------------  -------------
+SSTable format (all versions)   3.11(mc / md ), 2.2(la), 2.1.8 (ka)                    md format is supported in Scylla 4.3, 2021.1.0 and later
+------------------------------  -----------------------------------------------------  -------------
+JMX                             3.11                                                   More Below
+------------------------------  -----------------------------------------------------  -------------
 Configuration (cassandra.yaml)	3.11
-------------------------------  ---------------------------------------------  -------------
+------------------------------  -----------------------------------------------------  -------------
 Log                             NC
-------------------------------  ---------------------------------------------  -------------
+------------------------------  -----------------------------------------------------  -------------
 Gossip and internal streaming	NC
-------------------------------  ---------------------------------------------  -------------
+------------------------------  -----------------------------------------------------  -------------
 SSL                             NC
```

Comment:
@elcallio, is our TLS support incompatible with Cassandra's?

Reply:
TLS support for CQL, should be the same (I even added cql-pytest tests for that, run "test/cql-pytest/run --ssl" and it runs all the regular tests but connects to CQL with TLS). But if the heading is on some other use of SSL somewhere (maybe internal communication?) then I don't know.